### PR TITLE
Import export

### PR DESCRIPTION
**Pull request by engelke**
Minimal tests for importKey and exportKey. Only parameters that should cause the operation to succeed are tested; failure modes are not tested.

### CODE REVIEW

**Comment by sideshowbarker** [comment]
@jimsch needs review

File: WebCryptoAPI/import_export/ec_importKey.js [hunk 1]
```diff
@@ -0,0 +1,279 @@
+// Test importKey and exportKey for non-PKC algorithms. Only "happy paths" are
```
Comment (jimsch) [inline WebCryptoAPI/import_export/ec_importKey.js]: Is this comment correct?  I assume that non-PKC means non-Public Key Crypto, but this is a PKC file

Comment (engelke) [inline WebCryptoAPI/import_export/ec_importKey.js]: No, the comment is wrong. It's now been fixed, both here and for RSA.

**Comment by engelke** [comment]
I can't figure out how to fix the conflicts non-destructively, so I'm going to kill this PR, create a new branch, add files to it, and then submit a new PR.

**Pull request closed without merging**